packages/dashboard/src/components/tasks/task-alert.tsx

```diff
@@ -287,7 +287,7 @@ export function TaskAlertDialog({ alert, removeAlert }: TaskAlertDialogProps): J
         onDismiss={removeAlert}
         acknowledgedBy={taskAlert.acknowledgedBy}
         onAcknowledge={taskAlert.acknowledgedBy ? undefined : acknowledgeAlert}
-        onInspect={() => setOpenTaskInspector(true)}
+        // onInspect={() => setOpenTaskInspector(true)}
```

Comment:
Maybe we can add a comment about why we comment on this line in this iteration.
Or simply delete it if it will no longer be used.

Reply:
https://github.com/open-rmf/rmf-web/pull/845/commits/dd8190de2092ce6df7a26088558a584c22a8118f